lib/main.js

```diff
@@ -5,14 +5,31 @@ const parseParams = require('./utils').parseParams
 
 function Busboy (opts) {
   if (!(this instanceof Busboy)) { return new Busboy(opts) }
-  if (opts.highWaterMark !== undefined) { WritableStream.call(this, { autoDestroy: false, highWaterMark: opts.highWaterMark }) } else { WritableStream.call(this, { autoDestroy: false }) }
+
+  if (typeof opts !== 'object') {
```

Comment:
can we also move limit validation here?

Reply:
not necessary, as node is already throwing an error by node.
added unit test anyway ;)